src/main/java/org/reficio/p2/P2Mojo.java

```diff
@@ -133,6 +133,16 @@
     private boolean skipInvalidArtifacts;
```

Comment:
Is it a Maven phase? But yes, it will be more clear to say it is during the (re)bundling of an artifact.